src/odemis/driver/semnidaq.py

```diff
@@ -576,7 +576,10 @@ def __init__(self, sem: AnalogSEM, scanner: "Scanner"):
         self._ao_task = None
         self._ao_data_next_sample = 0
         self._ao_data = None
-        self._min_ao_buffer_n = self._scanner.get_hw_buf_size()
+        # It seems that if the buffer used is <= hw buffer, the events are not triggered
+        # Though, it's not very clear if that's just coincidence, because we actually ask for events
+        # buffer size / 2
```

Comment:
Could you update the comment to make it explicit that this is why you add the `+ 1`